ci-operator/step-registry/baremetalds/e2e/ovn/bgp/pre/baremetalds-e2e-ovn-bgp-pre-commands.sh

```diff
@@ -122,34 +171,36 @@ oc apply -f - <<EOF
 apiVersion: frrk8s.metallb.io/v1beta1
 kind: FRRConfiguration
 metadata:
-  name: receive-filtered
+  name: receive-all
   namespace: openshift-frr-k8s
 spec:
   bgp:
     routers:
     - asn: 64512
       neighbors:
-      - address: 192.168.111.1
+      - address: 192.168.111.3
         asn: 64512
         disableMP: true
         toReceive:
           allowed:
-            mode: filtered
-      - address: fd2e:6f44:5dd8:c956::1
+            mode: all
```

Comment:
Please note that this will prevent to do any effective testing related to the filtered toReceive routes of the FRRConfiguration that OVNK adds. So it would be better to filter here just what you need.